.github/workflows/ompi_mpi4py_tests.yaml

```diff
@@ -9,15 +9,15 @@ on:
   workflow_call:
     inputs:
       env_name:
-        required: true
+        required: false
         type: string
 
 jobs:
   mpi4py-tests:
     runs-on: ubuntu-latest
     timeout-minutes: 30
     env:
-      ${{ inputs.env_name}}: 1
+      ${{ inputs.env_name || 'MAKE_TODAY_AN_OMPI_DAY' }}: 1
```

Comment:
Same as above.. now that env_name is not required, it would make sense to give it a default value, instead of `|| 'MAKE_TODAY_AN_OMPI_DAY'` everywhere.

Reply:
In this particular case, as `workflow_call` is the only trigger, you are definitely right. Fixed.